don't change /etc/dropbear to root-only
(avoids problems when booting with nfsroot)

diff --git a/package/network/services/dropbear/files/dropbear.init b/package/network/services/dropbear/files/dropbear.init
--- a/package/network/services/dropbear/files/dropbear.init
+++ b/package/network/services/dropbear/files/dropbear.init
@@ -100,8 +100,8 @@ keygen()
 	mkdir -p /etc/dropbear
 	mv /tmp/dropbear/dropbear_* /etc/dropbear/
 	lock -u /tmp/.switch2jffs
-	chown root /etc/dropbear
-	chmod 0700 /etc/dropbear
+	#chown root /etc/dropbear
+	#chmod 0700 /etc/dropbear
 }
 
 start_service()
